examples/step-68/step-68.cc

```diff
@@ -278,7 +236,7 @@ namespace Step68
     MappingQ1<dim>                             mapping;
     LinearAlgebra::distributed::Vector<double> velocity_field;
 
-    Vortex<dim> velocity;
+    Functions::RayleighKotheVortex<dim> velocity;
```

Comment:
Can you also make the same change in the performance benchmark while we are at it?
:)